src/thunderbots/software/ai/hl/stp/action/chip_action.cpp

```diff
@@ -55,19 +57,19 @@ void ChipAction::calculateNextIntent(IntentCoroutine::push_type& yield)
     //                   |        \ /
     //                   >         A       <
     //                                     |
-    //                                     | DIST_TO_FRONT_OF_ROBOT / 2
+    //                                     | 0 dist (Point A is where the ball is)
     //                                     |
     //                             O       |
     //             ball ->        O O      <
     //                             O
     //
     //                             |
     //                             V
-    //                     direction of chip
+    //                     direction of kick
 
     do
     {
-        // A vector in the direction opposite the chip (behind the ball)
+        // A vector in the direction opposite the kick (behind the ball)
```

Comment:
This is the `chip_action`, so this should still be:
```suggestion
        // A vector in the direction opposite the chip (behind the ball)
```
??

Reply:
Yup, my bad

Reply:
fixed